packages/cloud-cognitive/src/components/InlineEditV2/_inline-edit-v2.scss

```diff
@@ -65,7 +67,8 @@
     outline: none;
   }
 
-  .#{$block-class}-readonly .#{$block-class}__text-input.#{$carbon-text-input} {
+  .#{$block-class}-readonly .#{$block-class}__text-input.#{$carbon-text-input},
+  .#{$block-class}-readonly .bx--btn.bx--btn--icon-only.bx--tooltip__trigger {
```

Comment:
```suggestion
  .#{$block-class}-readonly .#{$carbon-prefix}--btn.#{$carbon-prefix}--btn--icon-only.#{$carbon-prefix}--tooltip__trigger {
```

Reply:
blah good catch. i always forget that haha